source/fundamentals/dates-and-times.txt

```diff
@@ -40,6 +40,15 @@ time into MongoDB:
    explicitly returns the current time in UTC. Avoid using the ``datetime.datetime.now()``
    method, with no arguments, which returns the current local time.
 
+How Can I Save a ``datetime.date`` Instance?
+-----------------------------------------------
+
+{+driver-short+} doesn't support saving ``datetime.date`` instances, since
+there is no BSON type for dates without times. Rather than having the
+driver enforce a convention for converting ``datetime.date``
+instances to ``datetime.datetime`` instances for you, you should
+perform any conversion in your client code.
```

Comment:
To make Vale happy:

```suggestion
there is no BSON type for dates without times. Perform any conversions in your client code rather than having the
driver enforce a convention for converting ``datetime.date``
instances to ``datetime.datetime`` instances for you.
```